lib/routes.jsx

```diff
@@ -115,6 +115,8 @@ var routeElements = Object.keys(pages).map(function(path) {
   return <Route path={path} component={pages[path]} key={path}/>;
 });
 
+var makerPartyLocales = ['de', 'fr', 'es', 'pl', 'it', 'nl', 'cs', 'bg'];
```

Comment:
@ScottDowne it was totally a thing :) 215,642(6.42%) of users requested `es-es` and `fr-FR` was set on the users browser when they visit donate repo, but because the snippet dealt with that that's why we didn't have any issue with it.